sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -95,30 +125,40 @@ TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
 
 credential.update("{new_key}");
 ```
-##### **Option 2**: Create TextAnalyticsClient with Azure Active Directory Credential
-To use an [Azure Active Directory (AAD) token credential][aad_credential],
-provide an instance of the desired credential type obtained from the [azure-identity][azure_identity] library.
-Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain] 
-name for your resource in order to use this type of authentication.
+##### Create a Text Analytics client using Microsoft identity platform (formerly Azure Active Directory)
```

Comment:
is this language we are using across all libraries now?
if so, there are still some references to AAD in your Readme like line 132 "Authentication whit AAD requires..."

Reply:
hmmm reading more into it, I am confused. Sounds more like we are talking about using our Azure identity package.
am I understanding this correctly?

Reply:
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/eventhubs/azure-messaging-eventhubs#create-an-event-hub-client-using-microsoft-identity-platform-formerly-azure-active-directory

Reply:
I am trying to align within java repo as well. So since Eventhub has been GAed. I mimic the same pattern here.

Reply:
How are other libraries doing it? Looks like guidance is to use DefaultCredential (which u are using). Still, name sounds confusing. It is not like AAD was renamed, right? it is more the approach we are suggesting is to use DefaultCredential which includes the logic to do AAD.

Reply:
If an user want to use AAD, 'Identity' is required to be included as dependency.